docs/_content/ontwikkelaars/notificeren.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Notificeren"
+date: '27-03-2019'
+weight: 110
+---
+
+Als component moet je events publiceren naar een notificatiecomponent (NC), of
+je wil kunnen acteren op events. Deze guide helpt je op weg.
+
+Als uitgangspunt wordt de [gehoste notificatiecomponent](https://ref.tst.vng.cloud/nc/)
+gebruikt.
```

Comment:
Vervangt dit nu [dit document](https://github.com/VNG-Realisatie/gemma-zaken/blob/3557f1285592f59ecde14fd90c608f359e0f02cf/docs/_content/ontwikkelaars/algemeen/notificaties.md), waar ook nog een [Pull Request](https://github.com/VNG-Realisatie/gemma-zaken/pull/846/files) voor open staat?

Wellicht naar verwijzen naar het document? Of alles absorberen wat nog nuttig is en doc verwijderen.

Reply:
Eh nee, was bedoeld als "we maken gebruiken van dit NC als NC" i.t.t. "we gaan zelf een NC opzetten volgens deze api spec".